.github/workflows/test.yml

```diff
@@ -42,3 +42,8 @@ jobs:
       run: |
         composer run setup-local-tests
         composer test
+    - name: Run WP Acceptance
+      if: matrix.php-versions == '7.2'
+      run: |
+        if [ -n "${{ secrets.AWS_ACCESS_KEY }}" ]; then ./vendor/bin/wpsnapshots configure 10up --aws_key=${{ secrets.AWS_ACCESS_KEY }} --aws_secret=${{ secrets.SECRET_ACCESS_KEY }} --user_name=Travis --user_email=travis@10up.com; fi
```

Comment:
@jeffpaul We'll need AWS_ACCESS_KEY and SECRET_ACCESS_KEY added to the repo secrets for the WP Acceptance snapshot. I adapted this line from Distributor, which uses Travis and passes Travis user credentials, though it should work the same in the GH action.